### PR TITLE
[Workspace] Register four get started cards in home page

### PR DESCRIPTION
**Pull request by yubonluo**
### Description

Register four get started cards in home page.

### Issues Resolved
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/7332

## Screenshot

image
image
image

## Testing the changes

1. Clone the latest osd code and run yarn osd bootstrap
2. Modify config/opensearch_dashboards.yml
```
workspace.enabled: true 
uiSettings:
  overrides:
    "home:useNewHomePage": true
```
3. Run `yarn start --no-base-path`
4. Click workspace picker

## Changelog

- feat: [Workspace] Register four get started cards in home page

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/plugins/workspace/public/plugin.ts [hunk 1]
```diff
@@ -372,7 +381,43 @@ export class WorkspacePlugin implements Plugin<{}, {}, WorkspacePluginSetupDeps>
     return {};
   }
 
-  public start(core: CoreStart) {
+  private registerGetStartedCardToNewHome(
+    core: CoreStart,
+    contentManagement: ContentManagementPluginStart
+  ) {
+    // console.log('------');
+    // debugger;
```
Comment (ruanyl) [inline src/plugins/workspace/public/plugin.ts:389]:
```suggestion
```

**Review by ruanyl — APPROVED** [review]
Thanks for address the issue so quickly, it looks good to me now :)

**Review by SuZhou-Joe — APPROVED** [review]

File: src/plugins/workspace/public/plugin.ts (hunk 1 above)
Comment (SuZhou-Joe) [inline src/plugins/workspace/public/plugin.ts:389]: Seems the comment out lines are not removed.

Comment (yubonluo) [reply src/plugins/workspace/public/plugin.ts]: Updated

**Review by SuZhou-Joe — APPROVED** [review]

**Review by ruanyl — APPROVED** [review]

**Review by SuZhou-Joe — APPROVED** [review]

**Pull request merged**